src/test/java/org/folio/services/kafka/topic/KafkaTopicsExistsTest.java

```diff
@@ -1,38 +1,79 @@
 package org.folio.services.kafka.topic;
 
-import io.vertx.core.Vertx;
+import static org.folio.utility.KafkaUtility.startKafka;
+import static org.folio.utility.KafkaUtility.stopKafka;
+import static org.folio.utility.RestUtility.TENANT_ID;
+import static org.folio.utility.VertxUtility.getVertx;
+import static org.folio.utility.VertxUtility.removeTenant;
+import static org.folio.utility.VertxUtility.startVertx;
+import static org.folio.utility.VertxUtility.stopVertx;
+
 import io.vertx.ext.unit.TestContext;
 import io.vertx.ext.unit.junit.VertxUnitRunner;
 import io.vertx.kafka.admin.KafkaAdminClient;
 import io.vertx.kafka.admin.NewTopic;
+import java.util.List;
+import java.util.Locale;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+import org.apache.kafka.common.errors.TopicExistsException;
 import org.folio.kafka.KafkaConfig;
 import org.folio.kafka.services.KafkaEnvironmentProperties;
-import org.folio.rest.api.StorageTestSuite;
-import org.folio.rest.api.TestBase;
+import org.folio.postgres.testing.PostgresTesterContainer;
+import org.folio.rest.persist.PostgresClient;
 import org.junit.After;
+import org.junit.AfterClass;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import java.util.List;
-
 @RunWith(VertxUnitRunner.class)
-public class KafkaTopicsExistsTest extends TestBase {
-  final short REPLICATION_FACTOR = 1;
+public class KafkaTopicsExistsTest {
+  private static final short REPLICATION_FACTOR = 1;
+
+  private KafkaAdminClient kafkaAdminClient;
+
+  @BeforeClass
+  public static void beforeAll()
+      throws InterruptedException,
+      ExecutionException,
+      TimeoutException {
 
-  KafkaAdminClient kafkaAdminClient;
+    // tests expect English error messages only, no Danish/German/...
+    Locale.setDefault(Locale.US);
+
+    PostgresClient.setPostgresTester(new PostgresTesterContainer());
+    startKafka();
+    startVertx(TENANT_ID);
```

Comment:
Why does this test need to deploy and enable the module?

Reply:
I wish I knew.

This test was previously doing this before by its extending of `TestBase`.
When this gets called and is not called via the `StorageTestSuite`, then the `invokeStorageTestSuiteAfter` boolean gets set appropriately and the `StorageTestSuite` methods are called outside of the Suite tests.

This is a major cause of some of the random, inconsistent, or confusing test failures.
I decided not to address what I would consider a design problem with the test due to the scope and focus of my changes. The PR is big enough as it is.

In every place where we have to call `startKafka();` that looks more like a unit test or is not inside the `StorageTestSuite` should be considered for redesign. I do not know what the redesign would look like. I only know that the fact that I have to do this to preserve previous behavior (without having to call StorageTestSuite methods) is a problem.

Reply:
Fair enough, this one can wait.